doc/api/process.markdown

```diff
@@ -873,6 +883,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to its
+parent process using `process.send()`. Each will be received as a
+['message'](child_process.html#child_process_event_message)
```

Comment:
Oh okay. Actually I received a comment in that PR, so I thought that that is the standard which we follow.